.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI Checks
+
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: [dev]
+  workflow_dispatch:
+
+jobs:
+  complexity:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check complexity
+        uses: FreeRTOS/CI-CD-Github-Actions/complexity@main
+        with:
+          path: ./
+
+  spell-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone This Repo
+        uses: actions/checkout@v2
+      - name: Run spellings check
+        uses: FreeRTOS/CI-CD-Github-Actions/spellings@main
+        with:
+          path: ./
+
+  formatting:
+    runs-on: ubuntu-latest
```

Comment:
If the uncrustify config can be stored in this repository?
if so, can you include one that you find reasonable?

Reply:
Sure, I can do that.